File: tests/fixtures/sample.c
/* fx.c -- one small translation unit. */

#include <stddef.h>

/* Add returns the sum of a and b. C gets no `a` series: the
   declaration opens with a return type, so nothing names it. */
int add(int a, int b) {
    return a + b; /* a trailing comment */
}

const char *url = "http://example.com/not-a-comment";

File: tests/fixtures/sample.ts
// fx.ts -- one small module.

/**
 * Returns the sum of a and b.
 */
export function add(a: number, b: number): number {
  return a + b;  // a trailing comment
}

/** A type alias carries documentation too. */
export type Sum = (a: number, b: number) => number;

export const url = `http://example.com/not-a-comment`;
